### PR TITLE
Add iRobot

### PR DESCRIPTION
**Pull request by ronschaeffer**
**Issue:** closes #

**Similarweb rank:**47,912

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Hex derived from color picker on official PNG files https://media.irobot.com/media-kits?item=17 and verified with SVG color here https://companieslogo.com/irobot/logo/

Cannot find official SVGs so based on SVG here https://companieslogo.com/irobot/logo/.

Chose iR mark instead of full wordmark since it is better suited to icon use. Company uses it in several place such as app icons https://play.google.com/store/apps/details?id=com.irobot.home and https://twitter.com/iRobot.

No guidelines found for TM use on iR mark. Company uses it inconsistently, e.g., on twitter but not app icons.

image

### CODE REVIEW

**Comment by adamrusted** [comment]
Thanks for this @ronschaeffer! There is actually a [base64-encoded](data:image/svg+xml;charset=utf-8;base64,PHN2ZyB3aWR0aD0nMTIwJyBoZWlnaHQ9JzI4JyBmaWxsPSdub25lJyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnPjxwYXRoIGQ9J00yOC4xNzggOC44MzNDMjkuMDYzIDQuNjU1IDI2LjY1Ny4yNSAyMS4zODguMjVIMTMuMTdsLTEuMzYyIDYuMzhoNi4xOTljMS4yNzEgMCAyLjA2NiAxLjA0NSAxLjc5MyAyLjI5NC0uMjcyIDEuMjcxLTEuNDk4IDIuMjQ4LTIuNzcgMi4yNDhoLTYuMjY3bDguNDAyIDE1LjUzaDguNzg3TDIyLjI3NCAxNi4yOGMxLjUyMi0uODE3IDQuOTczLTMuMTEgNS45MDQtNy40NDd6TTEuNDUzIDE5Ljg2N0wwIDI2LjcyNmg3LjQ5M2wzLjI3LTE1LjUzMWMtNC41NjQuMTU5LTguMTUyIDMuMjAxLTkuMzEgOC42NzN6TTYuMzggNy4wMzljMS45NTMgMCAzLjU0Mi0xLjU2NyAzLjU0Mi0zLjUyQzkuOTIyIDEuNTY3IDguMzMyIDAgNi4zOCAwIDQuNDI4IDAgMi44MzggMS41NjcgMi44MzggMy41MmMwIDEuOTMgMS41OSAzLjUxOSAzLjU0MiAzLjUxOXpNMTE0LjU5Ni4yNWgtNy4wODRsLTMuNjMzIDE3LjI4Yy0xLjA2NyA1LjA2MyAyLjU0MyA5Ljc0IDEwLjk4OSA5Ljc0LjA2OSAwIC4yMDUtLjAyMy4yMDUtLjAyM2wxLjQzLTYuNzY2Yy0yLjY3OS40MDktNi4xMDgtLjU5LTUuNDk0LTMuNDk3bC45MDgtNC4zMTRoNi4yNDRsMS4yOTQtNi4wNjJoLTYuMjQ0TDExNC41OTYuMjV6TTY5Ljc5NyAxNy4wMDdjLS41NDQgMi41NDMtMi42MzMgNC4zMTQtNS41ODUgNC4zMTRzLTQuMTEtMS44NjItMy41NDItNC40OTZjLjU2Ny0yLjYxMSAyLjQ1Mi00LjEzMiA1LjQwNC00LjEzMiAyLjkyOSAwIDQuMjY4IDEuNzcgMy43MjQgNC4zMTR6TTY3LjMgNi43NDRjLTEuNDA4IDAtMy44MzcuMzYzLTQuNjU1Ljc5NUw2NC4xOS4yNWgtNy4wNjFsLTMuNTY1IDE2Ljc1N2MtMS4yNzIgNS45MjYgMi43NDcgMTAuMjg2IDkuNDIzIDEwLjI4NiA3LjQ0NyAwIDEyLjM5Ny0zLjIwMiAxMy44MDUtOS44MzIgMS4zNC02LjQwMy0zLjM4My0xMC43MTctOS40OTEtMTAuNzE3em0yNy42NTYgMTAuMjYzYy0uNTQ1IDIuNTQzLTIuNTY2IDQuMzE0LTUuNTE4IDQuMzE0cy00LjIyMy0xLjc0OS0zLjY1Ni00LjM4M2MuNTY4LTIuNjEgMi41NDQtNC4yNDUgNS40OTUtNC4yNDUgMi45MyAwIDQuMjI0IDEuNzcgMy42NzkgNC4zMTR6bS0yLjQzLTEwLjI2M2MtNi42NzUgMC0xMi40NjUgMy45MjgtMTMuODA1IDEwLjI4NS0xLjM0IDYuMzM1IDIuNzkzIDEwLjI4NiA5LjMxIDEwLjI4NiA3Ljk5MiAwIDEyLjY2OS0zLjk1IDEzLjk4Ni0xMC4yODYgMS4zNC02LjM1Ny0yLjgxNi0xMC4yODUtOS40OTEtMTAuMjg1eicgZmlsbD0nIzZDQjg2QScvPjxwYXRoIGQ9J000NC41MDMgMTcuMDA3Yy0uNTQ0IDIuNTQzLTIuNTY1IDQuMzE0LTUuNTE3IDQuMzE0LTIuOTUyIDAtNC4yMjMtMS43NDktMy42NTYtNC4zODMuNTY4LTIuNjEgMi41NDMtNC4yNDUgNS40OTUtNC4yNDUgMi45MyAwIDQuMjIzIDEuNzcgMy42NzggNC4zMTR6TTQyLjA5NyA2Ljc0NGMtNi42NzYgMC0xMi40NjYgMy45MjgtMTMuODA2IDEwLjI4NS0xLjM0IDYuMzM1IDIuNzkzIDEwLjI4NiA5LjMxIDEwLjI4NiA3Ljk5MiAwIDEyLjY3LTMuOTUgMTMuOTg3LTEwLjI4NiAxLjMxNy02LjM1Ny0yLjgxNi0xMC4yODUtOS40OTEtMTAuMjg1em03Ni4yOTEtNS4yOTFoLS4zNDF2LjYxM2guMzQxYy4yNzIgMCAuMzg2LS4xMTMuMzg2LS4zMTggMC0uMTM2LS4wNjgtLjI5NS0uMzg2LS4yOTV6bS44NCAxLjU5aC0uMzRsLS41NjgtLjcyN2gtLjI5NXYuNzI3aC0uMjk1di0xLjg0aC42NThjLjUgMCAuNjgxLjI5Ni42ODEuNTQ1IDAgLjI5NS0uMTgxLjQ3Ny0uNDU0LjUyM2wuNjEzLjc3MnptLS44NjMuNDU0Yy43NSAwIDEuMzYzLS42MTMgMS4zNjMtMS4zNjMgMC0uNzQ5LS42MTMtMS4zNjItMS4zNjMtMS4zNjItLjc0OSAwLTEuMzYyLjYxMy0xLjM2MiAxLjM2MiAwIC43NS42MTMgMS4zNjMgMS4zNjIgMS4zNjN6bTAtMi45OTdjLjg4NiAwIDEuNjM1Ljc0OSAxLjYzNSAxLjYzNCAwIC45MDktLjcyNiAxLjYzNS0xLjYzNSAxLjYzNWExLjYyOCAxLjYyOCAwIDAgMS0xLjYzNS0xLjYzNWMwLS44ODUuNzI3LTEuNjM0IDEuNjM1LTEuNjM0eicgZmlsbD0nIzZDQjg2QScvPjwvc3ZnPg==) svg in their website header, which we should use as our starting point. I'll review shortly, but will need the source setting to their website regardless. It also appears you're having trouble with the linter, I've found if I've got any other document formatter than Prettier installed in my environment, the file will auto-format to add the newline in at the end.

**Comment by ronschaeffer** [comment]
OK. Thanks. Noted re the source setting.

I can't see the base64 SVG in the header. Am I just missing it?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7270,6 +7270,11 @@
             "hex": "25313C",
             "source": "https://www.iris.co.uk"
         },
+        {
+            "title": "iRobot",
+            "hex": "69BE55",
+            "source": "https://companieslogo.com/irobot/logo/"
```
Comment (adamrusted) [inline _data/simple-icons.json:7276]:
```suggestion
            "source": "https://www.irobot.com"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
```
url(data:image/svg+xml;charset=utf-8;base64,PHN2ZyB3aWR0aD0nMTIwJyBoZWlnaHQ9JzI4JyBmaWxsPSdub25lJyB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnPjxwYXRoIGQ9J00yOC4xNzggOC44MzNDMjkuMDYzIDQuNjU1IDI2LjY1Ny4yNSAyMS4zODguMjVIMTMuMTdsLTEuMzYyIDYuMzhoNi4xOTljMS4yNzEgMCAyLjA2NiAxLjA0NSAxLjc5MyAyLjI5NC0uMjcyIDEuMjcxLTEuNDk4IDIuMjQ4LTIuNzcgMi4yNDhoLTYuMjY3bDguNDAyIDE1LjUzaDguNzg3TDIyLjI3NCAxNi4yOGMxLjUyMi0uODE3IDQuOTczLTMuMTEgNS45MDQtNy40NDd6TTEuNDUzIDE5Ljg2N0wwIDI2LjcyNmg3LjQ5M2wzLjI3LTE1LjUzMWMtNC41NjQuMTU5LTguMTUyIDMuMjAxLTkuMzEgOC42NzN6TTYuMzggNy4wMzljMS45NTMgMCAzLjU0Mi0xLjU2NyAzLjU0Mi0zLjUyQzkuOTIyIDEuNTY3IDguMzMyIDAgNi4zOCAwIDQuNDI4IDAgMi44MzggMS41NjcgMi44MzggMy41MmMwIDEuOTMgMS41OSAzLjUxOSAzLjU0MiAzLjUxOXpNMTE0LjU5Ni4yNWgtNy4wODRsLTMuNjMzIDE3LjI4Yy0xLjA2NyA1LjA2MyAyLjU0MyA5Ljc0IDEwLjk4OSA5Ljc0LjA2OSAwIC4yMDUtLjAyMy4yMDUtLjAyM2wxLjQzLTYuNzY2Yy0yLjY3OS40MDktNi4xMDgtLjU5LTUuNDk0LTMuNDk3bC45MDgtNC4zMTRoNi4yNDRsMS4yOTQtNi4wNjJoLTYuMjQ0TDExNC41OTYuMjV6TTY5Ljc5NyAxNy4wMDdjLS41NDQgMi41NDMtMi42MzMgNC4zMTQtNS41ODUgNC4zMTRzLTQuMTEtMS44NjItMy41NDItNC40OTZjLjU2Ny0yLjYxMSAyLjQ1Mi00LjEzMiA1LjQwNC00LjEzMiAyLjkyOSAwIDQuMjY4IDEuNzcgMy43MjQgNC4zMTR6TTY3LjMgNi43NDRjLTEuNDA4IDAtMy44MzcuMzYzLTQuNjU1Ljc5NUw2NC4xOS4yNWgtNy4wNjFsLTMuNTY1IDE2Ljc1N2MtMS4yNzIgNS45MjYgMi43NDcgMTAuMjg2IDkuNDIzIDEwLjI4NiA3LjQ0NyAwIDEyLjM5Ny0zLjIwMiAxMy44MDUtOS44MzIgMS4zNC02LjQwMy0zLjM4My0xMC43MTctOS40OTEtMTAuNzE3em0yNy42NTYgMTAuMjYzYy0uNTQ1IDIuNTQzLTIuNTY2IDQuMzE0LTUuNTE4IDQuMzE0cy00LjIyMy0xLjc0OS0zLjY1Ni00LjM4M2MuNTY4LTIuNjEgMi41NDQtNC4yNDUgNS40OTUtNC4yNDUgMi45MyAwIDQuMjI0IDEuNzcgMy42NzkgNC4zMTR6bS0yLjQzLTEwLjI2M2MtNi42NzUgMC0xMi40NjUgMy45MjgtMTMuODA1IDEwLjI4NS0xLjM0IDYuMzM1IDIuNzkzIDEwLjI4NiA5LjMxIDEwLjI4NiA3Ljk5MiAwIDEyLjY2OS0zLjk1IDEzLjk4Ni0xMC4yODYgMS4zNC02LjM1Ny0yLjgxNi0xMC4yODUtOS40OTEtMTAuMjg1eicgZmlsbD0nIzZDQjg2QScvPjxwYXRoIGQ9J000NC41MDMgMTcuMDA3Yy0uNTQ0IDIuNTQzLTIuNTY1IDQuMzE0LTUuNTE3IDQuMzE0LTIuOTUyIDAtNC4yMjMtMS43NDktMy42NTYtNC4zODMuNTY4LTIuNjEgMi41NDMtNC4yNDUgNS40OTUtNC4yNDUgMi45MyAwIDQuMjIzIDEuNzcgMy42NzggNC4zMTR6TTQyLjA5NyA2Ljc0NGMtNi42NzYgMC0xMi40NjYgMy45MjgtMTMuODA2IDEwLjI4NS0xLjM0IDYuMzM1IDIuNzkzIDEwLjI4NiA5LjMxIDEwLjI4NiA3Ljk5MiAwIDEyLjY3LTMuOTUgMTMuOTg3LTEwLjI4NiAxLjMxNy02LjM1Ny0yLjgxNi0xMC4yODUtOS40OTEtMTAuMjg1em03Ni4yOTEtNS4yOTFoLS4zNDF2LjYxM2guMzQxYy4yNzIgMCAuMzg2LS4xMTMuMzg2LS4zMTggMC0uMTM2LS4wNjgtLjI5NS0uMzg2LS4yOTV6bS44NCAxLjU5aC0uMzRsLS41NjgtLjcyN2gtLjI5NXYuNzI3aC0uMjk1di0xLjg0aC42NThjLjUgMCAuNjgxLjI5Ni42ODEuNTQ1IDAgLjI5NS0uMTgxLjQ3Ny0uNDU0LjUyM2wuNjEzLjc3MnptLS44NjMuNDU0Yy43NSAwIDEuMzYzLS42MTMgMS4zNjMtMS4zNjMgMC0uNzQ5LS42MTMtMS4zNjItMS4zNjMtMS4zNjItLjc0OSAwLTEuMzYyLjYxMy0xLjM2MiAxLjM2MiAwIC43NS42MTMgMS4zNjMgMS4zNjIgMS4zNjN6bTAtMi45OTdjLjg4NiAwIDEuNjM1Ljc0OSAxLjYzNSAxLjYzNCAwIC45MDktLjcyNiAxLjYzNS0xLjYzNSAxLjYzNWExLjYyOCAxLjYyOCAwIDAgMS0xLjYzNS0xLjYzNWMwLS44ODUuNzI3LTEuNjM0IDEuNjM1LTEuNjM0eicgZmlsbD0nIzZDQjg2QScvPjwvc3ZnPg==)
```

Available in the as the background-image of the element `a.logo-home.s-header__logo-home` on [this page](https://www.irobot.co.uk/)

**Comment by ronschaeffer** [comment]
Thanks!

I updated the iR mark in the SVG to base it on logo SVG you found on www.irobot.com. The geometry was indeed slightly different to my original.

**Comment by ronschaeffer** [comment]
I can't figure out the lint failure. The error is the an extraneous unexpected newline in irobot.svg. I'm tried adding and removing newlines several times. In the test check log, there is one version of the SVG that ends in \n.

**Comment by adamrusted** [comment]
Thanks for your attempts on this @ronschaeffer - unfortunately if you have a formatter baked into your IDE, or you use GitHub directly, it adds the newline in. We've got a Prettier config file at the root of our repository, so I tend to just enable that for this particular project.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -7289,6 +7289,11 @@
             "hex": "25313C",
             "source": "https://www.iris.co.uk"
         },
+        {
+            "title": "iRobot",
+            "hex": "69BE55",
```
Comment (adamrusted) [inline _data/simple-icons.json:7294]:
```suggestion
            "hex": "6CB86A",
```

**Review by adamrusted — APPROVED** [review]
My editor is showing that as the main colour of the logo, and that marries up with what is against `--color-primary` on the website stylesheet. Other than that, the SVG is a perfect match!

I'll merge the hex change in and get this merged into Dev, thanks @ronschaeffer!

**Pull request merged**